frameworks/kafka/universe/config.json

```diff
@@ -990,6 +990,12 @@
           "default": 1,
           "minimum": 1
         },
+        "ssl_endpoint_identification_algorithm": {
+          "title": "ssl.endpoint.identification.algorithm",
```

Comment:
Note to selves: let's use a title like "Enable SSL" and link to the UI issue JIRA.